Tools/ApplicationDebugger/jacobi/README.md

```diff
@@ -384,11 +368,15 @@ success; the relative error (9.97509e-05) is below the desired tolerance 0.0001
 #### GPU
 
 Start debugging GPU only after first two bugs are fixed on CPU.
+To debug on Intel GPU device, the device must be Level Zero.  The Level
+Zero GPU device, if available, is the default choice when no
+`ONEAPI_DEVICE_SELECTOR` is specified.  It corresponds to
+`ONEAPI_DEVICE_SELECTOR=level_zero:gpu`.
 
 Bug 3 is immediately hit while offloading to GPU:
 
 ```
-$ ./jacobi-bugged gpu
+$ ./jacobi-bugged
```

Comment:
Don't we need `ONEAPI_DEVICE_SELECTOR=level_zero:gpu` here? Or do we leave it because it's the default? Is there any guarantee that it will be the default?

Reply:
I was leaving it for the default choice. In the ONEAPI_DEVICE_SELECTOR doc it is said:

>The default choice will be one of these devices, usually preferring a Level Zero GPU device, if available.

I can add it here to be fully explicit and for the case, if no L0 GPU is available.